Description: Fix FTBFS with gcc-4.7
 Small header include change. This is borderlinde cosmetic, but still needed
 to prevent the FTBFS.
Author: Paul Tagliamonte <paultag@ubuntu.com>
Origin: vendor
Bug: https://code.google.com/p/wvstreams/issues/detail?id=34
Bug-Debian: http://bugs.debian.org/667418
Last-Update: 2013-10-30

--- a/utils/wvuid.cc
+++ b/utils/wvuid.cc
@@ -33,6 +33,7 @@
 
 #else // not WIN32
 
+#include <unistd.h>
 
 WvString wv_username_from_uid(wvuid_t uid)
 {
